partner_firstname/__openerp__.py

```diff
@@ -22,7 +22,9 @@
     'name': 'Partner first name and last name',
     'summary': "Split first name and last name for non company partners",
     'version': '8.0.2.1.0',
-    'author': "Camptocamp, Grupo ESOC, Odoo Community Association (OCA)",
+    "author": "Camptocamp, "
+              "Grupo ESOC Ingeniería de Servicios, "
```

Comment:
Their contribution is in https://github.com/OCA/partner-contact/pull/227, not merged yet.